README.md

```diff
@@ -159,3 +159,29 @@ For example, if `3000` is in use, `core` will run on `3001`.
 | Components Storybook | http://localhost:6006 |
 
 ![-----------------------------------------------------](https://storage.googleapis.com/bigcommerce-developers/images/catalyst_readme_hr.png)
+
+### Testing
```

Comment:
@chanceaclark I did not add instructions for visual regression tests as we earlier talked about moving those to running them on mac platform. Once that happens, it will be easy to put that down into instructions. Don't want to confuse and add lot of it into docs when it will be rarely looked at.